frontend/src/app/app-routing.module.ts

```diff
@@ -88,74 +88,69 @@ const routes: Routes = [
             children: [
               {
                 path: '',
-                data: {
-                  breadcrumb: 'overview',
-                  redirect: (data: Data) => `/project/${data.project?.slug}`,
-                },
+                data: { breadcrumb: undefined },
                 component: ProjectDetailsComponent,
               },
               {
                 path: 'metadata',
                 data: {
-                  breadcrumb: 'metadata',
+                  breadcrumb: 'Configure Project',
                   redirect: (data: Data) =>
                     `/project/${data.project?.slug}/metadata`,
                 },
                 component: EditProjectMetadataComponent,
               },
               {
                 path: 'models',
-                data: { breadcrumb: 'models' },
                 children: [
                   {
                     path: 'create',
-                    data: { breadcrumb: 'create' },
+                    data: { breadcrumb: 'Create Model' },
                     component: CreateModelComponent,
                   },
                 ],
               },
               {
                 path: 'model',
-                data: {
-                  breadcrumb: 'models',
-                  redirect: (data: Data) => `/project/${data.project?.slug}`,
-                },
                 children: [
                   {
                     path: ':model',
                     component: ModelWrapperComponent,
-                    data: {
-                      breadcrumb: (data: Data) => data.model?.name,
-                      redirect: (data: Data) =>
-                        `/project/${data.project?.slug}`,
-                    },
 
                     children: [
                       {
                         path: 'modelsources',
                         data: {
-                          breadcrumb: 'model sources',
+                          breadcrumb: 'Model Sources',
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/2f832994-ebb2-4378-9d55-bf9cba87e70d)

Here, the model slug is missing in the title. It's not clear for the user which model the model sources belong to.

Reply:
I added the model slug as intermediate breadcrumb forwarding back to the project page.